apps/frontend/src/libs/modules/api/base-http-api.ts

```diff
@@ -0,0 +1,123 @@
+import { type ContentType, ServerErrorType } from "~/libs/enums/enums.ts";
+import { configureString } from "~/libs/helpers/helpers.ts";
+import {
+	type HTTP,
+	type HTTPCode,
+	HTTPError,
+	HTTPHeader,
+} from "~/libs/modules/http/http.ts";
+import { type Storage, StorageKey } from "~/libs/modules/storage/storage.ts";
+import { type ServerErrorResponse, type ValueOf } from "~/libs/types/types.ts";
+
+import {
+	type HTTPApi,
+	type HTTPApiOptions,
+	type HTTPApiResponse,
+} from "./libs/types/types.ts";
+
+type Constructor = {
+	baseUrl: string;
+	path: string;
+	http: HTTP;
+	storage: Storage;
+};
+
+class BaseHttpApi implements HTTPApi {
```

Comment:
```suggestion
class BaseHTTPApi implements HTTPApi {
```
shouldn't be it as well?